app/src/main/res/values/strings.xml

```diff
@@ -444,8 +444,8 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="due_today">Due today</string>
     <string name="due_future">Due in future</string>
     <string name="completed">Completed</string>
-    <string name="match_any">Match Any</string>
-    <string name="match_all">Match All</string>
+    <string name="matching_any">Matching Any</string>
+    <string name="matching_all">Matching All</string>
```

Comment:
made is clearer that the _current_ state is matching all / any vs the state after you click the button

Reply:
Hm, depending on the language the translation can be even the same at both. For the english original I feel that the active (Match) is better than the passive (Matching), but that's a persona feeling.

Reply:
Hmmm. Perhaps we can use add a unicode check box?

`☐ Match all` vs `🗹 Match all`

Reply:
Ideally it would be an actual checkbutton, but modifying the Alerert dialog to achieve this seems too much work..

Reply:
no keep as string. active prefered. unicode might be a option, but not sure how it works out